pages/index.tsx

```diff
@@ -82,24 +72,20 @@ const IndexPage: React.FC<InferGetStaticPropsType<typeof getStaticProps>> = ({
         errorFilter
       ) as Synth<{ type: ContractType }>[]
   );
-  const { data: totalTvl } = useQuery(
-    "total tvl",
-    async () => await client.getLatestTvl()
-  );
+
   const { data: totalTvm } = useQuery(
     "total tvm",
     async () => await client.getLatestTvm()
   );
-  const { data: totalTvlChange } = useQuery(
-    "total tvl change",
-    async () => {
-      const [{ value: ydayTvl = NaN } = {}] = await client.request(
-        "global/globalTvlHistorySlice",
-        Math.floor(oneDayAgo().toSeconds())
-      );
-      return formatTvlChange(ydayTvl, totalTvl!);
-    },
-    { enabled: Boolean(totalTvl) }
+
+  const { data: defillamaTvl } = useQuery(
+    "defillama tvl",
+    async () => await getDefillamaTvl()
+  );
+
+  const { data: defillamaPercentChange } = useQuery(
```

Comment:
again if we just call these `totalTvlChange` and `totalTvl`, you can keep some of the logic after this unchanged

Reply:
Updated